### ISSUE TITLE
Unnecesary missing docblocks errors for typed properties in PHP 7.4

### ISSUE THREAD

**Issue by Pnoexz** [squizlabs/PHP_CodeSniffer#2775]
One of the new features of PHP 7.4 is the declaring the type of a property in classes, which can render the `@var` docblock unnecessary.

Example code:

```php
class SomeClass
{
    public string $someString;

    /**
     * Some helpful comment
     */
    public int $someInt;
}
```

Will trigger two errors: `Missing member variable doc comment (Squiz.Commenting.VariableComment.Missing)` on the string and `Missing @var tag in member variable comment (Squiz.Commenting.VariableComment.MissingVar)` on the int.

Ignoring both errors is not desirable because it will not report the errors on untyped properties.

**Comment by gsherwood** [comment]
This sniff enforces consistency because it's not a 7.4-only sniff.

If you just want to ignore a single error message, you can do that in a ruleset.xml file:
```xml
<rule ref="Squiz.Commenting.VariableComment.MissingVar">
  <severity>0</severity>
</rule>
```
Does that help?

**Comment by RSickenberg** [comment]
IMHO, the OG had a good point. Adding `@var` on an already typed property makes 0 sense, commenting to comment too. 

This reminds me this: 
![image](https://user-images.githubusercontent.com/16922548/145556613-45c8a268-514e-457f-b1aa-3944d2a4b253.png)

**Comment by voku** [comment]
I have the same problem, I want to force "@var" if there is no native type, so that other tools that depends on that can work. But if the native type is used and there is no comment after "@var <p>foo bar</p>", then it's only duplicate code.

**Comment by mbomb007** [comment]
Relevant for Drupal: https://www.drupal.org/project/coder/issues/3123282
